modes/_shared.md

```
@@ -25,22 +25,74 @@
+| Red flags | Blockers |
+| **Global** | Weighted average of ALL above |
+
+**Decision Logic:**
+- No early rejection gate.
+- 4.0+ is a good match; below 4.0 is lower priority.
+
+---
 
 **Score interpretation:**
-- 4.5+ → Strong match, recommend applying immediately
+- 4.5+ → Strong match, apply immediately
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)